iceoryx_dds/include/iceoryx_dds/dds/cyclone_data_reader.hpp

```diff
@@ -33,28 +33,68 @@ namespace dds
 class CycloneDataReader : public DataReader
 {
   public:
+    /// @brief delete default ctor
```

Comment:
In my opinion, it makes no sense to document deleted methods. They are not available for the user and why document something which is not present. 
But it would be helpful to document why they were deleted!

Reply:
That's fine. We weren't sure about it so it was brought here.